api/nodes/views.py

```diff
@@ -243,6 +359,62 @@ def perform_create(self, serializer):
         user = self.request.user
         serializer.save(creator=user)
 
+    # overrides ListBulkCreateUpdateDestroyAPIView
+    def bulk_update(self, request, *args, **kwargs):
+        """
+        Correctly formats bulk PUT/PATCH response
+        """
+        response = bulk_generics.ListBulkCreateUpdateDestroyAPIView.bulk_update(self, request, *args, **kwargs)
```

Comment:
This can just be a super call to the parent class; no need to hard-code a particular class this way.